static/bidder-info/conversant.yaml

```diff
@@ -1,6 +1,6 @@
 endpoint: "http://api.hb.ad.cpe.dotomi.com/cvx/server/hb/ortb/25"
 maintainer:
-  email: "CNVR_PublisherIntegration@conversantmedia.com"
+  email: "PublisherIntegration@epsilon.com"

```

Comment:
@AaronColbyPrice  
Thank you for your contribution to the Prebid server. An email is sent from prebid team to verify `PublisherIntegration@epsilon.com` email address. Please respond back with "received" message to complete verification

Reply:
@AaronColbyPrice  email has bounced back with following message. Please fix it. 

```
Your message to PublisherIntegration@epsilon.com couldn't be delivered.
The group PublisherIntegration only accepts messages from people in its organization or on its allowed senders list, and your email address isn't on the list.
```

Reply:
@onkarvhanumante excellent catch, we are testing now and it works internally, but not externally.  We are fixing now and I'll follow up.

Reply:
@AaronColbyPrice let us know when to verifiy email again

Reply:
Email verification is completed!